cmd/main.go

```diff
@@ -137,7 +129,19 @@ func loadConfig() config {
 		dbConfig: dbConfig,
 	}
 }
+func connectToNats(svc exapp.Service, logger logger.Logger) {
+
+	logger.Info(fmt.Sprintf("connecting %s", defNatsURL))
+
+	nc, err := nats.Connect(defNatsURL)
+	defer closeConn(nc, logger)
 
+	if err != nil {
+		logger.Error("Failed to connect to nats")
+	} else {
```

Comment:
Avoid `else`. If `err != nil` log, and `return`. Something like this: https://github.com/mainflux/mainflux/blob/master/cmd/normalizer/main.go#L51-L56

Reply:
In Go, in general avoidn `if-else` when `if` is sufficient. Otherwise, this if it is a good candidate for `switch`.